main.go

```diff
@@ -1026,8 +1052,12 @@ func (tf *transformer) transformCompile(args []string) ([]string, error) {
 		}
 	}
 
-	if tf.curPkgCache, err = loadPkgCache(tf.curPkg, tf.pkg, files, tf.info, ssaPkg); err != nil {
-		return nil, err
+	tf.curPkgCache, err = loadPkgCache(tf.curPkg)
+	if err != nil {
+		tf.curPkgCache, err = computePkgCache(tf.curPkg, tf.pkg, files, tf.info, ssaPkg)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
just like the GARBLE_CACHE change, I think this is remaining from the old preexec and should be reverted. Or if not, please explain.

Reply:
This one is needed I will explain

Reply:
I use `loadPkgCache` in `realForReflected`, where I want to ignore packages which are not cached yet instead of computing them.

This also makes it more explicit where the cache is actually computed, I found it a little confusing before.